meorg_client/client.py

```diff
@@ -496,6 +498,123 @@ def model_output_query(self, model_id: str) -> Union[dict, requests.Response]:
             url_params=dict(id=model_id),
         )
 
+    def model_output_update(
+        self, model_id: str, updated_fields: dict
+    ) -> Union[dict, requests.Response]:
+        """
+        Remove specific new model output entity
```

Comment:
Should this be "update"?

Reply:
See comment below

Reply:
I think Ben might be referring to the doc string:

```suggestion
        Update specific fields of an existing model output.
```